pyfr/integrators/base.py

```diff
@@ -53,13 +52,13 @@ def __init__(self, backend, systemcls, rallocs, mesh, initsoln, cfg):
         self.dtmin = 1.0e-12
 
         # Determine the amount of temp storage required by this method
-        nreg = self._stepper_nregs
+        self.nreg = self._stepper_nregs
```

Comment:
nregs needs to be passed to init_system and multip and normal integrator has different values. Now it's instance variable and passed as a kwarg, so it is consistent with the cfg treatment.